core/sql/sqlcomp/CmpSeabaseDDLroutine.cpp

```diff
@@ -348,20 +348,32 @@ void CmpSeabaseDDL::createSeabaseLibrary(
       processReturn();
       return;
     }
+
+  ComTdbVirtTableTableInfo * tableInfo = new(STMTHEAP) ComTdbVirtTableTableInfo[1];
```

Comment:
I just copied this from similar code in the DDL component.  If we decide to change this, then we should change all occurances.  Something to consider.